src/eureka/S4cal_StellarSpectra/s4cal_StellarSpec.py

```diff
@@ -0,0 +1,285 @@
+#! /usr/bin/env python
+
+# Generic Stage 4cal Calibrated Stellar Spectra pipeline
+
+import numpy as np
+import os
+import time as time_pkg
+from copy import deepcopy
+import astraeus.xarrayIO as xrio
+import scipy.interpolate as spi
+import matplotlib.pyplot as plt
+
+from eureka.version import version
+from eureka.lib import manageevent as me
+from eureka.lib import util, logedit, plots
+from eureka.S3_data_reduction.sigrej import sigrej
+from eureka.S4cal_StellarSpectra.s4cal_meta import S4cal_MetaClass
```

Comment:
This style of importing eureka components isn't consistent with what we've done everywhere else, and I think in theory it could cause headaches when using Eureka!'s code without installing it (which I often do when testing/tweaking things).

Reply:
Good catch!  I do the same.